packages/sign-client/src/controllers/engine.ts

```diff
@@ -258,9 +264,9 @@ export class Engine extends IEngine {
   public request: IEngine["request"] = async <T>(params: EngineTypes.RequestParams) => {
     this.isInitialized();
     await this.isValidRequest(params);
-    const { chainId, request, topic } = params;
-    const id = await this.sendRequest(topic, "wc_sessionRequest", { request, chainId });
-    const { done, resolve, reject } = createDelayedPromise<T>();
+    const { chainId, request, topic, expiry } = params;
```

Comment:
Thought: we need to make sure we document this param properly regarding the expected value always being in seconds. Otherwise I think we'll end up with confused devs who aren't clear on what unit should be passed for `expiry`.

This already trips us up internally and tbh I'd prefer if we use labelling like `FIVE_MINUTES_IN_SECONDS`, but that's a whole other refactoring story 😅

Reply:
Good idea. I tried to provide a descriptive error message in case an invalid value is provided and also updated it to include that the value is expected in seconds